discovery/file/file.go

```diff
@@ -49,6 +62,9 @@ var (
 
 func init() {
 	discovery.RegisterConfig(&SDConfig{})
+	prometheus.MustRegister(fileSDScanDuration)
```

Comment:
Thanks, can we register 3 at the same time? prometheus.MustRegister(fileSDScanDuration, fileSDReadErrorsCount, fileSDTimeStamp)

Reply:
Sure，By the way, update all in this package.